packages/sdk/src/actor/actor.ts

```diff
@@ -72,6 +72,7 @@ export interface ActorLike {
 	 * Any actors parented to this actor will also be exclusive to the given user.
 	 */
 	exclusiveToUser: Guid;
+	owner: Guid;
```

Comment:
When a client that owns an actor disconnects, how is ownership updated? Could it be the switch of authoritative peer picks them up in the client?

Reply:
Currently, whole topic around ownership and simulation authority is a big todo.
We'll have to address it more thoroughly. Disconnected client is not covered ATM, and it should next topic to deal.

Reply:
I am thinking if there is an easy fix to avoid broken state after first commit.